app/js/lib/foxbox/services/base.js

```diff
@@ -0,0 +1,147 @@
+'use strict';
+
+const TYPE = 'Unknown';
+
+// Private members.
```

Comment:
__nit:__ It's nitpicking, but seems ```// Private members.``` doesn't bring any value here :) With the following comments it's super clear what this is for :)